frontend/apps/reader/readerui.lua

```diff
@@ -277,19 +278,19 @@ function ReaderUI:init()
         end
         -- make sure we render document first before calling any callback
         self:registerPostInitCallback(function()
-            local start_ts = ffiUtil.getTimestamp()
+            local start_tv = TimeVal:now()
             if not self.document:loadDocument() then
                 self:dealWithLoadDocumentFailure()
             end
-            logger.dbg(string.format("  loading took %.3f seconds", ffiUtil.getDuration(start_ts)))
+            logger.dbg(string.format("  loading took %.3f seconds", (TimeVal:now() - start_tv):tonumber()))
```

Comment:
Could it make sense to have:
```lua
TimeVal:getDurationSince(start_tv)
TimeVal:getDurationSinceAsNumber(start_tv)
-- or we could just use this if we can't find a nice name for the previous one:
TimeVal:getDurationSince(start_tv):tonumber()
```

Reply:
The issue with this approach is that you don't know *which* clock source to compare to ;).

True, we generally use MONOTONIC, but doing something like that means we *could* feed it a start_tv in REALTIME, and then comparing to MONOTONIC, which is obviously wrong ;).

Since the "DIY" approach is barely any more characters to type, it's not really worth trying to figure out a way around that one ;).

Reply:
You could expect by contract that the value fed should have been gotten with TimeVal:now().

May be the DIY approach is natural for you who have been playing with TimeVal for months, but well, 10 times the same "DIY" trick, one could think we could have been clever to factorize it (like I initially did when introducing ffiUtil.getTimestamp/getDuration()).
(I have a few more of these benchmark log in my own patch :)

Reply:
That's a fair point ;).